grpc-utils/src/main/java/com/glegoux/grpc/server/GrpcServerSimpleArguments.java

```diff
@@ -77,20 +77,23 @@ private Option getPort(String name, String desc) {
         return Option.builder().longOpt(name)
             .hasArg()
             .required(false)
-            .type(Integer.class)
+            .type(Number.class)
```

Comment:
Certainly, linked to the bug!

Reply:
Because ` public static <T> T createValue(final String str, final Class<T> clazz) throws ParseException` is called in package `org.apache.commons.cli`: in the class `TypeHandler` there is 
`throw new ParseException("Unable to handle the class: " + clazz);` because `Integer.class` is not taken if the list of allowed class type.

Reply:
Good catch 🦸 !